pc-service/src/main/java/com/pokemaster/model/Team.java

```diff
@@ -33,6 +39,12 @@ public Team(int teamId, Trainer trainer, List<OwnedPokemon> pokemon) {
 		this.trainer = trainer;
 		this.pokemon = pokemon;
 	}
+
+	public Team() {
```

Comment:
Instead of the constructor, you could add:
```suggestion
	@NoArgsConstructor
```
at the top from Lombok